frontend/tests/components/dashboard/enrolled-droplets-grid.test.tsx

```
@@ -3,7 +3,7 @@ import { EnrolledDropletsGrid } from "@/components/dashboard/enrolled-droplets-g
 import { getCurrentUser } from "@/lib/auth/session";
 import { getAuthorizedUserByEmail } from "@/lib/requests/authorized-user";
 import {
-  getDropletAverageRating,
+  calculateDropletAverageRating,
```

ESLint

[error] 'calculateDropletAverageRating' is defined but never used.

(@typescript-eslint/no-unused-vars)